cmd/src/cli/run.rs

```diff
@@ -0,0 +1,47 @@
+use cli::{self, package};
+use error::DefaultResult;
+use holochain_container_api::{
+    config::{AgentConfiguration, Configuration, DNAConfiguration},
+    container::Container,
+};
+
+pub fn run(package: bool) -> DefaultResult<()> {
```

Comment:
Can you give us some /// rustdoc comments up here about this?

Reply:
sure